pages.de/common/gcc.md

```diff
@@ -0,0 +1,24 @@
+# gcc
+
+> Präprozessiert und kompiliert C und C++ Quellcodedateien und linkt diese anschließend zusammen.
+> Mehr Informationen: <https://gcc.gnu.org>.
+
+- Kompiliert mehrere Quellcodedateien zu einer ausführbaren Datei:
+
+`gcc {{Quelldatei1.c}} {{Quelldatei2.c}} -o {{Ausgabebinärdatei}}`
+
+- Erlaubt Warnungen und debug-Symbole in der Ausgabedatei:
+
+`gcc {{source.c}} -Wall -Og -o {{Ausgabebinärdatei}}`
+
+- Inkludiert Bibliotheken aus anderen Ordnern:
+
+`gcc {{Quelldatei.c}} -o {{Ausgabebinärdatei}} -I{{Pfad_zu_Headerdatei}} -L{{Pfad_zu_Bibliothek1}} -l{{Pfad_zu_Bibliothek2}}`
```

Comment:
We use `/` for paths:
```suggestion
`gcc {{Quelldatei.c}} -o {{Ausgabebinärdatei}} -I{{Pfad/zu/Headerdatei}} -L{{Pfad/zu/Bibliothek1}} -l{{Pfad/zu/Bibliothek2}}`
```

Reply:
there was no slashes in the english original

Reply:
https://github.com/tldr-pages/tldr/blob/master/pages/common/gcc.md

Reply:
@Abrynos Yeah, I know, but I like your change. and I believe we should fix the English page too. Maybe you can add the edition in this PR if you want.